src/components/MainView.js

```diff
@@ -25,6 +51,16 @@ function MainView({ children }) {
     setMenuPanelOpen(!compactMode)
   }, [compactMode])
 
+  if (preferenceOption.path) {
+    return (
+      <GlobalPreferences
```

Comment:
Will this appear even if there is a global error?

Reply:
not sure if i follow here, we handle the global errors [here](https://github.com/aragon/court-dashboard/blob/57a4000e3a7cff114a47d95e8a3ec9f26aa6ed02/src/App.js#L26)

Reply:
Yep—this question was to check whether we would be able to change the network settings in the global error.

For example, if I change one of the network errors, but then get a fatal global error, it would be ideal to reset the settings there. Otherwise, you're more or less stuck until someone from support manually intervenes to help you clear storage.